### PR TITLE
Rename monitoring tab to Observe

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes**: 
https://issues.redhat.com/browse/ODC-6088

**Description**: 
Rename monitoring tab and nav option to Observe

**Screen shots / Gifs for design review**: 
image

image

image

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by spadgett** [comment]
Won't this break users' bookmarks?

**Comment by kyoto** [comment]
Thanks @spadgett

I guess we should leave the URLs unchanged and just change the text?

**Comment by vikram-raj** [comment]
Yes, users will get the 404 not found page if th

**Comment by vikram-raj** [comment]
Yes, users will get the 404 not found page if they have bookmarked the monitoring URL. But @serenamarie125 is fine with this change.

File: frontend/packages/dev-console/console-extensions.json [hunk 1]
```diff
@@ -340,11 +340,11 @@
...
+      "id": "observe",
       "perspective": "dev",
       "section": "top",
-      "name": "%devconsole~Monitoring%",
-      "href": "/dev-monitoring",
+      "name": "%devconsole~Observe%",
+      "href": "/dev-observe",
       "dataAttributes": {
         "data-quickstart-id": "qs-nav-monitoring",
         "data-tour-id": "tour-monitoring-nav",
```
Comment (abhinandan13jan) [inline frontend/packages/dev-console/console-extensions.json:350]: should we not update the quickstart ids?

**Comment by abhinandan13jan** [comment]
@vikram-raj correct me if wrong, I see we need updated QuickStarts for `monitoring-sample-app` in the `console-operator` repo
and in integration test, global.ts do we need to update the following?
```
export enum devNavigationMenu {
  Add = '+Add',
  Topology = 'Topology',
  Monitoring = 'Monitoring',
  Builds = 'Builds',
  Search = 'Search',
  Helm = 'Helm',
  Project = 'Project',
  ProjectAccess = 'Project Access',
  Pipelines = 'Pipelines',
  ConfigMaps = 'Config Maps',
  Secrets = 'Secrets',
  GitOps = 'GitOps',
  Environments = 'Environments',
}
```

**Comment by abhinandan13jan** [comment]
Ran locally, runs fine

**Comment by vikram-raj** [comment]
> @vikram-raj correct me if wrong, I see we need updated QuickStart for `monitoring-sample-app` in the `console-operator` repo
> and in integration test, global.ts do we need to update the following?
> 
> ```
> export enum devNavigationMenu {
>   Add = '+Add',
>   Topology = 'Topology',
>   Monitoring = 'Monitoring',
>   Builds = 'Builds',
>   Search = 'Search',
>   Helm = 'Helm',
>   Project = 'Project',
>   ProjectAccess = 'Project Access',
>   Pipelines = 'Pipelines',
>   ConfigMaps = 'Config Maps',
>   Secrets = 'Secrets',
>   GitOps = 'GitOps',
>   Environments = 'Environments',
> }
> ```

Thanks @abhinandan13jan , yes we need to update the qucikstarts for the monitoring as well. I will open a PR.

**Comment by spadgett** [comment]
> Yes, users will get the 404 not found page if they have bookmarked the monitoring URL.

I dislike breaking links if we can avoid it. Is it possible to add a react router redirect? (I don't think this would work for tabs, though).

The URL content itself is not that important. Few users look at it, and many browsers even hide the path these days. It's more important that the URLs are stable IMO. Have we considered updating the labels in the UI and leaving the URLs as they are?

**Comment by serenamarie125** [comment]
Having a redirect would be my preference if possible.  If not possible, keeping the URLs as monitoring works.

**Comment by kyoto** [comment]
My concern is that if we change the UI wording again in the future or reorganize the sidebar, we could then be looking at changing the URLs again and adding another redirect. So it seems easier to leave the URLs as is.

The current `monitoring` in the URLs seems like a logical choice to me, even if the UI wording differs.

**Comment by vikram-raj** [comment]
Updated the text in quickstart.
Here is the PR https://github.com/openshift/console-operator/pull/563

**Pull request merged**